lib/v2/screens/profile_screens/profile/components/profile_bottom.dart

```diff
@@ -7,132 +7,163 @@ import 'package:seeds/i18n/profile.i18n.dart';
+                    children: [
+                      Expanded(
+                        child: Padding(
+                          padding: const EdgeInsets.all(16.0),
+                          child: Column(
+                            children: [
+                              Row(
                                 children: [
                                   Text(
-                                    'Resident'.i18n,
-                                    style: Theme.of(context).textTheme.headline6!.copyWith(color: AppColors.canopy),
+                                    'You are on the way from'.i18n,
+                                    style: Theme.of(context).textTheme.subtitle2,
                                   ),
-                                  Padding(
-                                    padding: const EdgeInsets.symmetric(horizontal: 5.0),
-                                    child: Container(
-                                      width: 32,
-                                      height: 32,
-                                      decoration: const BoxDecoration(
-                                        shape: BoxShape.circle,
-                                        color: Colors.white,
-                                      ),
-                                      child: Center(
-                                        child: Text(
-                                          'to'.i18n,
-                                          style:
-                                              Theme.of(context).textTheme.headline7.copyWith(color: AppColors.primary),
+                                ],
+                              ),
+                              Padding(
+                                padding: const EdgeInsets.symmetric(vertical: 8.0),
+                                child: Row(
+                                  children: [
+                                    Text(
+                                      'Resident'.i18n,
+                                      style: Theme.of(context).textTheme.headline6!.copyWith(color: AppColors.canopy),
+                                    ),
+                                    Padding(
+                                      padding: const EdgeInsets.symmetric(horizontal: 5.0),
+                                      child: Container(
+                                        width: 32,
+                                        height: 32,
+                                        decoration: const BoxDecoration(
+                                          shape: BoxShape.circle,
+                                          color: Colors.white,
+                                        ),
+                                        child: Center(
+                                          child: Text(
+                                            'to'.i18n,
+                                            style: Theme.of(context)
+                                                .textTheme
```

Comment:
same here? do we have styles?

Reply:
I only remain to fix this style will do it as soon as Jason adds it.

Reply:
In the meantime I'll do this merge.